### PR TITLE
Scholarship translations to ES and CA

### PR DESCRIPTION
**Pull request by angelmixu**
Here are both translations, please, take a look for any typo, or rewording.

I'm not very fond of "si els nostres assistens necessiten un assitent"... but I can't think anything similar :_D

### CODE REVIEW

File: 2019/scholarship-info.ca.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+### Sobre les nostres beques
+
+L'objectiu principal del RustFest sempre ha sigut unir a la comunitat. Per descomptat, _"Comunitat"_, incloeïx diferents grups, persones marginades, i totes aquelles persones que normalmente no es poden permetre assitir -- moltes de les nostres [polítiques](/diversity/ca/] estan anotades per garantir que proporcionem un event assequible i meticulós que pretén ser tant inclusiu i acollidor com humanament possible sigui.
+
+Completant totes les nostres altres polítiques, un altra cop aquest any estem desenvolupant un programa de Beques per fer un cop de mà als nostres membres de la comunitat que necessiten ajudar per assistir al RustFest. Instem a persones de moniries subrepresentades en tecnologia que sol·licitin aquesta ajuda. Això incloeïx, però no <span style="color:red"> **està** </span>, limitat a persones qeu pertanyen a qualsevol dels següent grups: persones de color, dones, persones de génere no binari i no declarat, minusvàlids, persones amb enfermetats mentals, i persones LGBTQIA+. La Beca ofereïx tant transport i allotjament, com accés complementari a la conferència.
```
Comment (piny4man) [inline 2019/scholarship-info.ca.md]: ojo aqui el TYPO , **qeu**

File: 2019/scholarship-info.ca.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+**Ambdòs participants, tant els que hagin reservat personalment el biajte, o a través de la conferència,** tindran la possibilitat d'enviar les despeses després de la conferència per cobrir/reemborsar diversos costs derivats de la seva assistència. Els segúents tickets es poden enviar per a reemborsament després de l'event:
+
+- despeses de viatje i allotjament (reemborsament finbs el límit previst)
+- despeses de sol·licitut del visat (reemborsament complet)
+- fins a 100€ en diverses despeses relacionades amb el viatje (despeses de viatje)
+
+Les despeses del biatje es poden fer servir per cobrir els viatje desde/fins a l'ambaixada (per la sol·licitut del visat) i a l'aeropot; adicionalment, per al transport local, menjar, medicació etc. a Barcelona, Per favor, tingues en compte que no es reemborsaran begudes alcoholiques. Per rebre el reemborsament de les despeses del visat, el demandant haurà de guardar el resguard del pagament del visat a l'ambaixada i enviar-lo.
```
Comment (piny4man) [inline 2019/scholarship-info.ca.md]: ojo aqui el TYPO, **biatge**

File: 2019/scholarship-info.ca.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+Les Beques del RustFest poden incloure ajudes per al viatje si es demanen. Entenem que, no tothom es pot permetre pagar de cop el cost del viatje fins al RustFest, inclús quan proporcionem reemborsament per a aquests costos, així que adicionalment oferim la opció de reservar el viatje en nom de l'agraciat.
+
+Pels agraciats que prefereixin reservar el seu propi viatje, podem oferir un reemborsament al alça basat en les expectatives del cost del viatje, per a que així no hi hagi sorpreses per a cap de les parts. Aquesta és una gran opció per als que prefereixin la flexibilitat/privacitat de reservar el seu propia viatje, i fins i tot ens pot fer estalviar diners i complicacions (que podriem fer servir per altres tarees i agraciats), però de totes maneres, aprofita les ventatjes en reserva de viatjes que ofereix la conferència si creus que ho necessites.
+
+**Ambdòs participants, tant els que hagin reservat personalment el biajte, o a través de la conferència,** tindran la possibilitat d'enviar les despeses després de la conferència per cobrir/reemborsar diversos costs derivats de la seva assistència. Els segúents tickets es poden enviar per a reemborsament després de l'event:
```
Comment (piny4man) [inline 2019/scholarship-info.ca.md]: typo **biatge**

Comment (piny4man) [reply 2019/scholarship-info.ca.md]: tambien **següents**

File: 2019/scholarship-info.ca.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+- despeses de viatje i allotjament (reemborsament finbs el límit previst)
```
Comment (piny4man) [inline 2019/scholarship-info.ca.md]: **fins**, se escapan letras del teclado jeje

File: 2019/scholarship-info.ca.md (hunk 2 above)
Comment (piny4man) [inline 2019/scholarship-info.ca.md]: ojo **biatje** otra vez

File: 2019/scholarship-info.es.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+Adicionalmente, disponemos de un número limitado de entradas gratuítas para personas que de otra manera no hubiesen podido permitirse atender a la conferencia y viven en Barcelona o cercanías.
+
+Si crees que te identificas con lo mencionado arriba, por favor [baja al final de esta página](#send-an-application) y envía una solicitud. Si no, por favor, ayúdanos a correr la voz, o [contribuyendo](#funding).
+
+
+### Fechas
+
+Las solicitudes se abrirán el 15 de agosto, y permanecerán abiertas hasta que todas las entradas hayan sido asignadas y el presupuesto exhaurido (pero almenos hasta el 15 de setiembre).
```
Comment (piny4man) [inline 2019/scholarship-info.es.md]: ep! exhaurido nop! es **agotado**

File: 2019/scholarship-info.es.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+Empezaremos a contactar a los agraciados a principios de setiembre, aunque continuaremos asignando entradas contínuamente hasta setiembre u octubre, hasta que nuestro presuuesto se acabe. Todos los que hayan pedido una beca de viaje recibirán una respuesta del estado de su solucitud hasta 4 semanas antes de la conferencia.
```
Comment (piny4man) [inline 2019/scholarship-info.es.md]: ojo que te sobra una U en **presupuesto**

File: 2019/scholarship-info.es.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+**Ambos participantes, tanto los que hayan reservado personalmente el viaje, o a través de la conferencia,** tendrán la posibilidad de enviar los gastos después de la conferencia para cubrir/reembolsar varios costes derivados de su asistencia. Los siguientes tickets se pueden enviar par reembolso después del evento:
+
+- costes de viaje y alojamiento (reembolso asta el límite previsto)
+- costes de solicitud de visado (reembolso completo)
+- hasta 100€ en varios gastos relacionados con el viaje (gastos de viaje)
+
+Los gastos de viaje se pueden usar para cubrir el viaje desde/hacia la embajada (para la solicutud del visado) y al aeropuerto; adicionalmente, para el transporte local, comida, medicación etc. en Barcelona. Por favor, ten en cuenta que no se reembolsarán bebidas alcoholicas. Para recibir el reembolso de los costes del visado, el demandante habrá de guardar el resguardo del pago de la visa en la embajada y enviarlo.
```
Comment (piny4man) [inline 2019/scholarship-info.es.md]: resguardo del pago de la visa --> comprobande del pago del visado

**Review by piny4man — CHANGES_REQUESTED** [review]
Hay algunos cambios a realizar

**Comment by flaki** [comment]
Hey folks, this looks amazing! Thank you for working late hours to get this out of the door, it's really much much appreciated!

Let me know when it all looks good and we can merge and I'll transition these over to the website and create the respective forms!

Thanks again!

File: 2019/scholarship-info.ca.md (hunk 1 above)
Comment (angelmixu) [reply 2019/scholarship-info.ca.md]: arreglado!

File: 2019/scholarship-info.ca.md (hunk 2 above)
Comment (angelmixu) [reply 2019/scholarship-info.ca.md]: fixed!

File: 2019/scholarship-info.ca.md (hunk 3 above)
Comment (angelmixu) [reply 2019/scholarship-info.ca.md]: hecho!

File: 2019/scholarship-info.ca.md (hunk 4 above)
Comment (angelmixu) [reply 2019/scholarship-info.ca.md]: ya ves, y mira que me lo repasé... xD

File: 2019/scholarship-info.ca.md (hunk 2 above)
Comment (angelmixu) [reply 2019/scholarship-info.ca.md]: joder... que sé que va con v... pero debo tener el dedo torcido xD

**Review by angelmixu — COMMENTED** [review]
I'd say I have fixed all the typos, we can probably merge now! :)

**Comment by angelmixu** [comment]
@flaki I don't know how to set that the requested changes has been made, but you can merge it when you please :)

**Pull request merged**